app/views/user_mailer/unlock_instructions.text.erb

```diff
@@ -1,5 +1,3 @@
-Hello <%= @resource.name %>,
```

Comment:
Same here. These are overridden devise templates though I see we have them copied into the local view path under `devise/mailers`. It might be worth checking these are even being used. Again, coverage here is not great I believe.